modules/KalturaSupport/resources/mw.KCuePoints.js

```diff
@@ -405,6 +409,36 @@
 			}
 			return foundAttr;
 		},
+		/**
+		 * if have same CP earlier - hide current CuePoint
+		 * @param  allCP - array of CP where try to find same CP
+		 * @param  currentCuePointIndex - position from current CP
+		 *
+		 */
+		validateSameCuePoints:function (allCP, currentCuePointIndex) {
+			var currentCP = allCP[currentCuePointIndex];
+			var prevCP = this.getPrevCPWithCorrectType(allCP,currentCuePointIndex-1);
+			if(prevCP !== false && currentCP){
+				var differentByCreatedAt = Math.abs(currentCP.createdAt - prevCP.createdAt);
+				var differentByStartTime = Math.abs(currentCP.startTime - prevCP.startTime);
+				var isTheSamePartnerData = currentCP.partnerData === prevCP.partnerData;
+				var isTheSameTags = currentCP.tags === prevCP.tags;
```

Comment:
@MaksymKorniiov if we remove this line it will also cover admin-cp.
when an admin-user switches to live he sends admin-tagged-start CP and immediately public CP and therefore admin-users sees 2 slides. Removing this should fix also admin duplicity